lib/Slack.cs

```diff
@@ -43,7 +43,7 @@ static class Slack
                     .Json
                     .JsonConvert
                     .DeserializeObject<SlackUserList>(responseContent)
-                    .Members.First(member => member.Id == userId).Name;
+                    .Members.First(member => member.Id == userId.Split('|')[0])?.Name ?? "";
```

Comment:
Added a split for when it resolves the errant use case described in PR. Didn't want it to just pass back null, but there's a better way to handle only returning valid usernames.